Tests/OnPropertyNameChangedConfigTests.cs

```diff
@@ -31,6 +32,31 @@ public void Default()
         Assert.True(moduleWeaver.InjectOnPropertyNameChanged);
     }
 
+    [Fact]
+    public void ModuleWeaver_WhenInjectOnPropertyNameChangedIsTrue_ThrowsWeavingException()
```

Comment:
this one seems to pass even without your change

Reply:
Yeah that's right as it verifies that the error is displayed when the Inject property is true.

Before the error would be displayed in both true & false.

I can remove this one if you like?

Reply:
Fair enough. Let’s keep it